boot.mount

```diff
@@ -0,0 +1,11 @@
+[Unit]
+ConditionKernelCommandLine=ostree
+
+[Mount]
+What=/sysroot/boot
+Where=/boot
+Type=none
+Options=bind
+
+[Install]
+WantedBy=local-fs.target
```

Comment:
Doesn't this need the `dh_systemd_enable boot.mount` in `debian/rules`? Or is the presence of `boot.mount` enough for systemd to not generate the mount unit?